tests/utils.py

```diff
@@ -37,4 +58,9 @@ def call_from_slack(
             body=body,
         )
 
-    return dict(slack_message=m.call_args[1]["message"], response=response)
+    return dict(slack_message=mock_request.post.call_args, response=response)
+
+
+def get_raw_block_text(slack_message):
+    blocks = slack_message[1]["json"]["blocks"]
+    return "".join([b.get("text", {}).get("text", "") for b in blocks])
```

Comment:
Having a hard time understanding this one. Might it be rewritten to make it more obvious what it does?